fmriprep/cli/run.py

```diff
@@ -56,16 +63,19 @@ def main():
     # CRITICAL Call build_workflow(config_file, retval) in a subprocess.
     # Because Python on Linux does not ever free virtual memory (VM), running the
     # workflow construction jailed within a process preempts excessive VM buildup.
-    with Manager() as mgr:
```

Comment:
I had a vague memory that the numpy OMP_NUM_THREADS turned out to be the problem that this was trying to resolve? Do we still need this?

Reply:
It was the main culprit. This could probably be removed - but it would be nice to check how much memory it builds up if made within the main thread - probably not much.